### PR TITLE
Refactor: import syntax in library part 1

### PR DESCRIPTION
**Pull request by ninaandrup**
Related to #3148

Done in this PR:

- [x]  Time/Instant.flix
- [x]  Add.flix
- [x]  BigInt.flix
- [x]  Char.flix
- [x]  Console.flix
- [x]  Div.flix
- [x]  Environment.flix

### CODE REVIEW

File: main/src/library/BigInt.flix [hunk 1]
```diff
@@ -95,7 +95,7 @@ namespace BigInt {
     /// A successful parse is wrapped with `OK(x)`, a parse failure is indicated by `Err(_)`.
     ///
     pub def fromString(s: String): Result[BigInt, String] = try {
-        import java.lang.String.strip();
+        import java.lang.String.strip(): String & Pure;
         import new java.math.BigInteger(String) as parseBigInt;
         Ok(s |> strip |> parseBigInt) as & Pure
```
Comment (ninaandrup) [inline main/src/library/BigInt.flix:100]: Is it possible to use the notation in a case like this?
I couldn't seem to figure out how, if it is even possible.

File: main/src/library/Time/Instant.flix [hunk 2]
```diff
@@ -36,26 +36,27 @@ namespace Time {
...
 
         ///
         /// Returns the given instant `x` plus `d` number of nano seconds.
         ///
         @Experimental
         pub def plus(x: Time.Instant, d: Int64): Time.Instant =
+            // TODO: refactor import.
             import java.time.Instant.plusNanos(Int64);
             let Time/Instant.Instant(t) = x;
             Time/Instant.Instant(t `plusNanos` d) as & Pure
```
Comment (ninaandrup) [inline main/src/library/Time/Instant.flix:62]: I'm not sure what to do with this one.
If I write
```
import java.time.Instant.plusNanos(Int64): Time.Instant & Pure;
```
It says `>> Unable to unify the types: 'Instant' and 'Time.Instant'.`.
And if I write
```
import java.time.Instant.minusNanos(Int64): Instant & Pure;
```
It says `>> Undefined type 'Instant'.`.

Any recommendations?

Comment (mlutze) [reply main/src/library/Time/Instant.flix:62]: I think you need to put the full Java type as the signature.

`import java.time.Instant.plusNanos(Int64): ##java.time.Instant & Pure;`

The Java `Instant` and the Flix `Instant` are different types.

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**